srcpkgs/coWPAtty/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cowpatty'
+pkgname=coWPAtty
+version=4.6
+revision=1
+wrksrc="cowpatty-${version}"
+build_style=gnu-makefile
+disable_parallel_build=1
+make_use_env=1
+LDFLAGS=" -I /usr/share/include/"
```

Comment:
the space at the start is not needed.

also why it includes stuff under /usr/share ?

Reply:
because otherwise it does not find stuff like #include <openssl/md5.h>
It is weird but I assume the makefile is somehow broken. Because it is clearly installed in the masterdir.